run-smoke-test.sh

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env bash
 
 set -ex
-rm ember-try*.tgz || true
-npm pack
-now=$(date +%s)
-mv ember-try*.tgz "ember-try-${now}.tgz"
-cd smoke-test-app && yarn upgrade "ember-try@file:../ember-try-${now}.tgz" && yarn install && "./${1}"
+cd smoke-test-app && npm i && npm i --save-dev "../" && "./${1}"
```

Comment:
🤔 I think this is the cause of the CI failures. I think we still need to `npm pack` and install the tarball

Reply:
`npm install <tarball file>` is a legit thing https://docs.npmjs.com/cli/install#synopsis

Reply:
I think we might need to consider doing a major version bump on this one because without this change it fails in NPM 4 for some reason: https://travis-ci.org/ember-cli/ember-try/builds/498796103?utm_source=github_status&utm_medium=notification

Reply:
Assuming it _actually_ works on npm 4, I'd be fine not running the smoke tests under it.